test/e2e/apps/cronjob.go

```diff
@@ -104,8 +104,10 @@ var _ = SIGDescribe("CronJob", func() {
 		framework.ExpectNoError(err, "Failed to create CronJob in namespace %s", f.Namespace.Name)
 
 		ginkgo.By("Ensuring no jobs are scheduled")
-		err = waitForNoJobs(ctx, f.ClientSet, f.Namespace.Name, cronJob.Name, false)
-		framework.ExpectError(err)
+		err = framework.Gomega().
+			Consistently(ctx, framework.GetObject(f.ClientSet.BatchV1().CronJobs(f.Namespace.Name).Get, cronJob.Name, metav1.GetOptions{})).
+			Should(gomega.HaveField("Status.Active", gomega.BeEmpty()))
+		framework.ExpectNoError(err)
```

Comment:
This is not a `test/e2e/framework/*` helper package. Therefore you can directly call `gomega.Consistently` and then don't need to check for an error.